src/main/java/sparksoniq/jsoniq/compiler/translator/expr/postfix/extensions/DynamicFunctionCallExtension.java

```diff
@@ -48,6 +49,11 @@ public DynamicFunctionCallExtension(List<Expression> arguments, ExpressionMetada
         return getDescendantsFromChildren(result, depthSearch);
     }
 
+    @Override
+    public void initHighestExecutionMode() {
+        this._highestExecutionMode = ExecutionMode.LOCAL;
```

Comment:
Is it not the default already, i.e., is this not superfluous?

Reply:
Postfix extension (parent class of DynamicFunctionCallExtension) overrides init method to the behavior which is shared by all postfix extensions except dynamicFunctionCallExtension. This is always local, which is the behavior identical to the grandparent.

Reply:
I have added a comment to the code as well